src/nvme/mi.c

```diff
@@ -531,8 +539,8 @@ static void nvme_mi_admin_init_resp(struct nvme_mi_resp *resp,
 }
 
 static void nvme_mi_control_init_req(struct nvme_mi_req *req,
-				    struct nvme_mi_control_req *control_req,
-				    __u8 opcode, __u16 cpsp)
+				     struct nvme_mi_control_req *control_req, __u8 opcode,
+				     __u16 cpsp)
```

Comment:
The original formatting looks saner in my opinion. Let's keep it this way, please.

In an ideal world we would have used something like clang formatter from the beginning but I think this ship has sailed and we live in an imperfect world. And I really think checkpatch.pl is really a crutch, it helps for new submissions to minimize the review work but for the existing code I'd rather leave as it is unless it is completely offensive to read (e.g. the plugin code used to be very bad). Though the mi.c file is one of the very nicely formatted files and Jeremy is really taking care it stays this way.